aidants_connect_web/templates/public_website/home_page.html

```diff
@@ -7,6 +7,9 @@
 {% endblock extracss %}
 
 {% block content %}
+{% if infolettre_inscription %}
```

Comment:
Instinctivement j'aurais tendance à gérer ça avec la feature [`messages`](https://docs.djangoproject.com/en/3.0/ref/contrib/messages/#using-messages-in-views-and-templates) de Django, qu'on utilise du reste déjà pas mal, que penses-tu d'harmoniser cela ? :)

Reply:
j'ai modifié ! d'ailleurs j'en ai appris d'avantage sur les messages django, par exemple qu'ils sont passés automatiquement au template, du coup vraiment besoin de mettre `"messages": messages` dans le render des views ?

Reply:
Il me semble effectivement que ça n'est pas nécessaire, à partir du moment où [le bon _context processor_ est activé dans les `settings`](https://docs.djangoproject.com/en/3.0/ref/contrib/messages/#enabling-messages) du projet (les [_context processors_](https://docs.djangoproject.com/en/3.0/ref/templates/api/#writing-your-own-context-processors) étant de très simples fonctions Python chargées d'injecter systématiquement des valeurs supplémentaires dans le `dict` passé au moteur de templating).